pkg/cmd/admin/policy/modify_scc.go

```diff
@@ -22,6 +23,17 @@ const (
 	RemoveSCCFromUserRecommendedName  = "remove-scc-from-user"
 )
 
+const (
+	addSCCToUserExample = `  # Add restricted to user1 and user2
+  $ %[1]s restricted user1 user2
+
+  # Add privileged to serviceaccount1
```

Comment:
"Add the 'privileged' security context constraint to the service account 'serviceaccount1' in the current namespace"